src/um/winuser.rs

```diff
@@ -6597,7 +6597,7 @@ pub const EVENT_CONSOLE_START_APPLICATION: UINT = 0x4006;
 pub const EVENT_CONSOLE_END_APPLICATION: UINT = 0x4007;
 #[cfg(target_arch = "x86_64")]
 pub const CONSOLE_APPLICATION_16BIT: LONG = 0x0000;
```

Comment:
this should be point_width = "64"

content from header

#if defined(_WIN64)
#define CONSOLE_APPLICATION_16BIT       0x0000
#else
#define CONSOLE_APPLICATION_16BIT       0x0001
#endif